backend/env.json

```diff
@@ -53,5 +53,13 @@
   "EHR_RXFILL_URL": {
     "type": "string",
     "default": "http://localhost:8080/test-ehr/script/rxfill"
+  },
+  "USE_INTERMEDIARY": {
+    "type": "boolean",
+    "default": true
+  },
+  "INTERMEDIARY_ETASU_MET": {
+    "type": "string",
+    "default": "http://localhost:3003/4_0_0/GuidanceResponse/$rems-etasu"
```

Comment:
Why not just make this INTERMEDIARY_FHIR_URL and then add on the /GuidanceResponse/$rems-etasu this this as well as what is read from the table in doctorOrders.js.

Reply:
I suppose I could refactor the getRemsAdminEtasuUrl function so it finds the ETASU url if the appropriate environment variable is set. Is `GuidanceResponse/$rems-etasu` a fixed string from the REMS IG or does it follow a typical FHIR naming standard for endpoints? Or is that just specific to the `@projecttacoma/node-fhir-server-core` package?